test/sensitivities/functional/functional.jl

```diff
@@ -36,62 +33,48 @@ using DiffRules: diffrule, hasdiffrule
             s = broadcast(f, x_, y_)
             o = oneslike(unbox(s))
             ∇s = ∇(s, o)
-            ∇x = broadcast((z, z̄, x, y)->∇(f, Arg{1}, nothing, z, z̄, x, y),
-                           unbox(s), o, x, y)
-            ∇y = broadcast((z, z̄, x, y)->∇(f, Arg{2}, nothing, z, z̄, x, y),
-                           unbox(s), o, x, y)
+#            ∇x = sum(broadcast((z, z̄, x, y)->∇(f, Arg{1}, nothing, z, z̄, x, y), unbox(s), o, x, y))
+#            ∇y = broadcast((z, z̄, x, y)->∇(f, Arg{2}, nothing, z, z̄, x, y), unbox(s), o, x, y)
```

Comment:
@willtebbutt  is these test important i commented it out for some reason

Reply:
I think the problem is they pass in nothing as the value rule that was returned for preprocess which is not the case now because we actually use that value.
preprocess does all the work. ∇ just selects the right position
So can I delete them?